manifests/0000_90_dns-operator_03_prometheusrules.yaml

```diff
@@ -34,3 +34,15 @@ spec:
           severity: warning
         annotations:
           message: "CoreDNS is returning SERVFAIL for {{ $value | humanizePercentage }} of requests."
+      - alert: CoreDNSForwardHealthcheckFailureCount
+        annotations:
+          message: '{{ $value }} A CoreDNS counter for the number of failed health checks for an upstream server on {{ $labels.instance }}'
+        expr: coredns_forward_healthcheck_failure_count_total > 0
+        labels:
+          severity: warning
+      - alert: CoreDNSForwardHealthcheckBrokenCount
+        annotations:
+          message: '{{ $value }} A CoreDNS counter for the number of health check failures for all upstream servers on {{ $labels.instance }}'
```

Comment:
I think the message for both of these alerts should be more along the lines of alert information, rather than generic metric information. See the messages used for the other alerts in this file. Alert messages typically aren't read by a cluster administrator until an alert is firing.